src/rpc/server.cpp

```diff
@@ -169,7 +169,7 @@ std::string CRPCTable::help(const std::string& strCommand, const JSONRPCRequest&
     {
         const CRPCCommand *pcmd = command.second;
         std::string strMethod = pcmd->name;
-        if ((strCommand != "" || pcmd->category == "hidden") && strMethod != strCommand)
+        if ((strCommand != "" || pcmd->category == RPCCategory::hidden) && strMethod != strCommand)
```

Comment:
This is the volta stone of this refactoring: avoiding a strange behavior in the semantics of category identifiers.